src/components/TodoList/TodoList.tsx

```diff
@@ -16,85 +30,58 @@
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+      {todos.map(todo => (
+        <tr
+          key={todo.id}
+          data-cy="todo"
+          className={cn({
+            'has-background-info-light':
+              selectedTodo !== null && selectedTodo.id === todo.id,
+          })}
+        >
+          <td className="is-vcentered">{todo.id}</td>
+          {todo.completed ? (
+            <td className="is-vcentered">
+              <span className="icon" data-cy="iconCompleted">
+                <i className="fas fa-check" />
+              </span>
+            </td>
+          ) : (
+            <td className="is-vcentered" />
+          )}
+          <td className="is-vcentered is-expanded">
+            <p
+              className={cn({
+                'has-text-success': todo.completed,
+                'has-text-danger': !todo.completed,
+              })}
+            >
+              {todo.title}
+            </p>
+          </td>
+          <td className="has-text-right is-vcentered">
+            <button
+              onClick={() => {
+                onTodoSelect(todo);
+                onUserIdSelect(todo.userId);
+              }}
+              data-cy="selectButton"
+              className="button"
+              type="button"
+            >
+              <span className="icon">
+                <i
+                  className={cn({
+                    'far fa-eye': selectedTodo === null,
+                    'far fa-eye-slash':
+                      selectedTodo !== null && selectedTodo.id === todo.id,
+                  })}
```

Comment:
Good job on using the `classnames` library to conditionally apply classes. This makes your code cleaner and easier to read.